src/PointwiseFunctions/AnalyticSolutions/RelativisticEuler/RotatingStar.cpp

```diff
@@ -70,12 +75,39 @@ CstSolution::CstSolution(const std::string& filename,
       cst_file >> radius_[index] >> cos_theta_[index] >>
           rest_mass_density_[index] >> alpha_[index] >> rho_[index] >>
           gamma_[index] >> omega_[index] >> fluid_velocity_[index];
-      radius_[index] *= pow(equilibrium_kappa, polytropic_index_ * 0.5);
-      rest_mass_density_[index] *= pow(equilibrium_kappa, -polytropic_index_);
+      if (is_polytrope) {
+        radius_[index] *= pow(polytropic_constant, 0.5 * polytropic_index_);
+        rest_mass_density_[index] *=
+            pow(polytropic_constant, -polytropic_index_);
+        omega_[index] *= pow(polytropic_constant, -0.5 * polytropic_index_) /
+                         equatorial_radius_;
+        fluid_velocity_[index] *=
+            pow(polytropic_constant, -0.5 * polytropic_index_) /
+            equatorial_radius_;
+      } else {
+        radius_[index] *= sqrt(kappa_norm) / hydro::units::cgs::length_unit;
```

Comment:
Could you add a comment as to broadly speaking what's happening here, this is taking the data in the RotNS format and converting it to geometric units, right?  Perhaps add a pointer to wherever you got these conversion expressions from, I think it's mostly clear modulo the factors of 10^15 (which I think I understand), and the factors of kappa_norm (which I don't think I understand)

Reply:
Just pushed a change that added a comment clarifying this and probably addressing your other comments a bit. Let me know if it's clearer now.